source/access/mfa_guide.rst

```diff
@@ -238,10 +259,20 @@ what to do for MobaXTerm, PuTTY and NX:
   - Select 'Use key-based authentication with a SSH agent'
   - Click 'Modify' and verify that 'Forward authentication' is checked
 
+- FileZilla
```

Comment:
For FileZilla, we do have a dedicated page: https://docs.vscentrum.be/data/transfer/filezilla.html
I strongly recommend to remove the text here and only refer to that page. Else, we shall be maintaining two separate pages!

Reply:
This one you take with Maxime. I follow what he and Wouter have discussed. For now I will leave it.